src/main/java/mekanism/common/item/gear/ItemAtomicDisassembler.java

```diff
@@ -314,18 +349,40 @@ public DisassemblerMode byIndex(int index) {
             return MathUtils.getByIndexMod(MODES, index);
         }
 
+        // This is kinda hacky. A couple duplicates here to allow the different interfaces to play nicely.
         @Override
         public String getTranslationKey() {
             return langEntry.getTranslationKey();
         }
 
+        // Required by IRadialSelectorEnum
+        @Override
+        public ITextComponent getTextComponent() { return langEntry.translate(color); }
+
+        // Required by IRadialSelectorEnum.
+        // Used for radial menu identifiers
+        @Override
+        public ITextComponent getShortText() {
+            return this.getTextComponent();
+        }
+
+        // Efficiency of block destruction
         public int getEfficiency() {
             return efficiency;
         }
 
-        @Override
+        // The old interface required an override here. Left it here so that this would be easier to revert if necessary.
```

Comment:
This isn't good, we need to make a "disableable" version of the radial mode system as the atomic disassembler allows for disabling specific modes on the atomic disassembler.